### PR TITLE
Update variables.rst

### PR DESCRIPTION
**Pull request by FH-DESIGN**
I found the User_guide is quite old and provide some updates.

### CODE REVIEW

File: doc/source/User_guide/variables.rst [hunk 1]
```diff
@@ -29,8 +29,9 @@ access the ``variable_manager`` object for a more comprehensive set of functions
         >>> hfss["$PrjVar3"] = "34 * $PrjVar1/$PrjVar2"
         >>> hfss["$PrjVar3"]
         2.13628300444106
-        >>> hfss["$PrjVar3"].value
+        >>> hfss["$PrjVar3"].value # This line does not work in *.py file, pls check line #34 instead
         2.13628300444106
+        >>> hfss.evaluate_expression(hfss["$PrjVar3"])
```
Comment (MaxJPRey) [inline doc/source/User_guide/variables.rst:34]:
```suggestion
```

File: doc/source/User_guide/variables.rst [hunk 2]
```diff
@@ -29,8 +29,9 @@ access the ``variable_manager`` object for a more comprehensive set of functions
         >>> hfss["$PrjVar3"] = "34 * $PrjVar1/$PrjVar2"
         >>> hfss["$PrjVar3"]
         2.13628300444106
-        >>> hfss["$PrjVar3"].value
+        >>> hfss["$PrjVar3"].value # This line does not work in *.py file, pls check line #34 instead
```
Comment (MaxJPRey) [inline doc/source/User_guide/variables.rst:32]:
```suggestion
        >>> hfss.evaluate_expression(hfss["$PrjVar3"])
```

**Comment by MaxJPRey** [comment]
Thanks @FH-DESIGN for contributing to PyAEDT and submitting a Pull Request. I made some suggestions based on your feedback. Feel free to accept them or decline them.
Thanks again for contributing to PyAEDT.

File: doc/source/User_guide/variables.rst
Comment (dcrawforAtAnsys) [inline doc/source/User_guide/variables.rst:1]: LGTM. Thanks.

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**